test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import javax.rmi.CORBA.UtilDelegate;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void testIsAnyNull() {
+        assertTrue(Utils.isAnyNull(null, "Hello", 123, new Integer(10)));
+    }
+    
+    @Test
+    public void testIsAnyNotNull() {
+        assertFalse(Utils.isAnyNull("hello",123,new Integer(20)));
+    }
+
+    @Test
+    public void testElementsAreUnique() {
+        List<Integer> testVals = new ArrayList<Integer>();
+       
+        for(int i = 0; i < 5; i++){
+            testVals.add(i);
+        }
```

Comment:
You can use `List<Integer> list = Arrays.asList(1, 1, 1, 2)` to create and initialize a list in one line.